### PR TITLE
Feature/Succeed complex attributes

### PR DESCRIPTION
**Pull request by Milena-Czierlinski**

### CODE REVIEW

File: test/modules/attributes/AttributesController.test.ts [hunk 1]
```diff
@@ -1076,7 +1077,301 @@ describe("AttributesController", function () {
...
+                let version1ChildValues: string[]
+                let repoVersion0: LocalAttribute
+                let repoVersion1Params: IAttributeSuccessorParams
+                beforeEach(async function () {
+                    version0ChildValues = ["AStreet", "AHouseNo", "AZipCode", "ACity", "DE"]
+
+                    const identityAttribute = IdentityAttribute.from({
+                        value: {
+                            "@type": "StreetAddress",
+                            recipient: "ARecipient",
```
[the remark this answers is not in the document]
Comment (Milena-Czierlinski) [reply test/modules/attributes/AttributesController.test.ts:1091]: The value type of `recipient` is just a string. For all other properties, a dedicated child attribute is created, but for `recipient` not. Thus, we can't execute the tests for the child attributes (iterating over `versionXOfChildValues`) for `recipient`.

[the remark this answers is not in the document]
Comment (sebbi08) [reply test/modules/attributes/AttributesController.test.ts:1091]: understand

**Review by jkoenig134 — CHANGES_REQUESTED** [review]
blocking merge until monday

**Comment by Milena-Czierlinski** [comment]
Continued in https://github.com/nmshd/runtime/pull/7

**Pull request closed without merging**